DEVELOPMENT.md

```
@@ -31,14 +31,17 @@ Note: Jitar is managed as a monorepo with Lerna. You don't need specific knowled
 1. Update the [documentation](./documentation/README.md) if necessary.
 1. Run `npm run build` to build all packages.
 1. Run `npm run review` to run all tests and linting.
+1. Run `npm run changeset` to create a changeset for your changes.
 1. Commit and push the changes.
 1. Create a pull request.
+1. [CodeRabbit](https://www.coderabbit.ai/) will review your pull request automatically. It is very nitpicky, so don't get discouraged by the number of comments. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)